back/app/models/idea_image.rb

```diff
@@ -20,8 +20,19 @@
 #  fk_rails_...  (idea_id => ideas.id)
 #
 class IdeaImage < ApplicationRecord
+  attr_accessor :skip_image_presence
+
   mount_base64_uploader :image, IdeaImageUploader
   belongs_to :idea, inverse_of: :idea_images
 
+  validate :image_presence, unless: :skip_image_presence
   validates :idea, presence: true
+
+  private
+
+  def image_presence
+    return if image.present?
+
+    errors.add(:image, 'cannot be blank!')
+  end
```

Comment:
If we do go for this approach, and we wish to generalize it other similar image models, we could move the `attr_accessor` and the `image_presence` method to the parent class, `ApplicationRecord`, to reduce code duplication.